packages/libraries/main/src/matcher/dictionary/variants/matching/l33t.ts

```diff
@@ -55,7 +55,7 @@ class MatchL33t {
     )
     let hasFullMatch = false
     let isFullSubstitution = true
-    subbedPasswords.reverse().forEach((subbedPassword) => {
+    subbedPasswords.forEach((subbedPassword) => {
```

Comment:
The `reverse()` was providing some performance improvements.  At least with the strings that i tested as most of the time the substitution were matched with one of the later entries.

Reply:
I guess it was because the fully substituted passwords were the last ones.
Now getCleanPasswords returns them first, so the list is already reversed there.